scrapy_azure_exporter/azure_exporter.py

```diff
@@ -90,24 +96,33 @@ def _store_in_thread(self, file):
     def parse_azure_uri(cls, uri):
         "Takes in uri and extracts container name and filename"
         extracted_params = {}
+
         try:
             parsed_url = urlparse(uri)
             # split till 2nd occurrence of slash,
-            # so we can get container name and file name seperated.
-            splitted = parsed_url.path.split("/", 2)
+            # so we can get container name and file name separated.
+            # NOTE: A rsplit is necessary for compatibility with
+            # Azurite that have the path in the format
+            # `/<account_name>/<container_name>/<filename.ext>`
```

Comment:
```suggestion
            # `/<account_name>/<container_name>/<filename.ext>`
```

Does the proposed solution work for `/<account_name>/<container_name>/<folder>/<filename.ext>`?

Reply:
Please, check the new solution.